openapi/internal/rsoac/rsoac_test.go

```diff
@@ -0,0 +1,29 @@
+package rsoac
+
+import (
+	"testing"
+)
+
+func Test_Regex(t *testing.T) {
+	tests := []testConverterRegex{
+		{
+			`/OK/`,
+			`{
+        		"type": "string",
+        		"pattern": "OK"
+			}`,
+		},
+		{
+			`/^[A-Z][a-z]*( [A-Z][a-z]*)*$/`,
+			`{
+        		"type": "string",
+        		"pattern": "^[A-Z][a-z]*( [A-Z][a-z]*)*$"
+			}`,
+		},
+	}
+	for _, data := range tests {
```

Comment:
It would be nice to add another test with various, complex regex characters. Such as slash and others.

Reply:
Also, please add another test for the SchemaObject interface in schema_object_test.go.

And add a test of all methods from the SchemaInformer interface to informer_test.go (see func Test_Informer_RSchema(...))